package.json

```diff
@@ -1,14 +1,14 @@
 {
-  "name": "checker-framework-language-server",
-  "description": "LSP client for the Checker Framework",
+  "name": "checker-framework-vscode",
+  "description": "LSP client for the EISOP Checker Framework",
   "author": "eisopux",
   "license": "MIT",
-  "version": "0.2.0",
+  "version": "0.3.0",
   "repository": {
     "type": "git",
     "url": "https://github.com/eisopux/checker-framework-vscode.git"
   },
-  "publisher": "zhangjiangqige",
+  "publisher": "eisopux",
```

Comment:
Figure out whether this can be the Github organization name or whether it needs to be a user name.

Reply:
@wmdietl I think we can directly use eisop for the publisher. https://code.visualstudio.com/api/working-with-extensions/publishing-extension#create-a-publisher